### PR TITLE
[BACKLOG-36863] Page Zoom (AA) - Data Source Wizard Dialog

### PR DESCRIPTION
**Pull request by varuntangirala**
@pentaho/wcag , Please review
Depends on: https://github.com/pentaho/pentaho-commons-gwt-modules/pull/929

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 32s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
:ok_hand: All tests passed!
 
Tests run: **18**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/xul/job/PR-244/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by soagarwal1 — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 39s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
:ok_hand: All tests passed!
 
Tests run: **18**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/xul/job/PR-244/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 36s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
:ok_hand: All tests passed!
 
Tests run: **18**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/xul/job/PR-244/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 40s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
:ok_hand: All tests passed!
 
Tests run: **18**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/xul/job/PR-244/5/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 46s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
:ok_hand: All tests passed!
 
Tests run: **18**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/xul/job/PR-244/6/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: gwt/impl/src/main/java/org/pentaho/ui/xul/gwt/tags/GwtToolbarbutton.java [hunk 1]
```diff
@@ -148,12 +148,6 @@ public void init( com.google.gwt.xml.client.Element srcEle, XulDomContainer cont
       setVisible( "true".equals( vis ) );
     }
 
-    // Add class name to toolbar button
-    String className = srcEle.getAttribute( "pen:classname" );
-    if ( className != null && !className.isEmpty() ) {
-      ( (ToolbarButton) getManagedObject() ).addClassName( className );
-    }
-
```
[the remark this answers is not in the document]
Comment (dcleao) [reply gwt/impl/src/main/java/org/pentaho/ui/xul/gwt/tags/GwtToolbarbutton.java:156]: Thanks, @davidmsantos90! Fixed, in this PR and the GWT one, https://github.com/pentaho/pentaho-commons-gwt-modules/pull/929.

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 39s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
:ok_hand: All tests passed!
 
Tests run: **18**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/xul/job/PR-244/7/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by davidmsantos90 — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 44s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
:ok_hand: All tests passed!
 
Tests run: **18**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/xul/job/PR-244/9/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 2m
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
:ok_hand: All tests passed!
 
Tests run: **18**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/xul/job/PR-244/10/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 52s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 44s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
:ok_hand: All tests passed!
 
Tests run: **18**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/xul/job/PR-244/13/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 44s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
:ok_hand: All tests passed!
 
Tests run: **18**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/xul/job/PR-244/15/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 37s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 1m 17s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl core,gwt/impl
```
---
:ok_hand: All tests passed!
 
Tests run: **232**, Failures: **0**, Skipped: **175** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/xul/job/PR-244/17/testReport)

 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 33s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
:ok_hand: All tests passed!
 
Tests run: **18**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/xul/job/PR-244/18/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by soagarwal1 — APPROVED** [review]

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 40s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
:ok_hand: All tests passed!
 
Tests run: **18**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/xul/job/PR-244/19/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 31s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl gwt/impl
```
---
:ok_hand: All tests passed!
 
Tests run: **18**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/xul/job/PR-244/20/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by davidmsantos90 — APPROVED** [review]

**Pull request merged**